frontend/src/app/signUp/IdentityVerification/types.d.ts

```diff
@@ -9,3 +9,17 @@ interface Question {
 interface Answers {
   [outWalletAnswer: string]: string;
 }
+
+interface IdentityVerificationRequest {
+  firstName: string;
+  middleName?: string | null;
+  lastName: string;
+  dateOfBirth: string;
+  email: string;
+  phoneNumber: string;
+  streetAddress1: string;
+  streetAddress2?: string | null;
+  city: string;
+  state: string;
+  zip: string;
```

Comment:
I think there's one additional (optional) field that should be included here, `poBoxNumber`. https://github.com/CDCgov/prime-simplereport/compare/bmaden-usds/experian-initial#diff-eb07562bc641e9d536015163dc976db650e0672037116ff47e33b861ed97bba4R25 (this is the latest Experian backend work, for reference)

Reply:
Yeah I left it out because the Figma design didn't have it, but I did see that the backend type included it. Isn't it pretty standard to put PO Box as Street Address 1 for any address though? I was a little confused about that.

Reply:
It is definitely standard but the Experian request has both street2 and po box 😆   So we've included both on the backend.